### ISSUE TITLE
Post six new FAQs: Artificial intelligence and CC licenses

### ISSUE THREAD

**Issue by peterspdx** [creativecommons/creativecommons.org#929]
Instructions and text sent by email. For posting March 13. Posted here for tracking.

**Comment by kgodey** [comment]
This is done. I made the changes locally, copied them manually to the server (per `README.md`) and created a PR https://github.com/creativecommons/faq/pull/22 for @TimidRobot to review.

I ran into an issue where the site was cached in an unfortunate state (the table of contents had updated but the actual FAQs were not showing up), so I logged into Cloudflare and manually purged the cache for the `https://www.creativecommons.org/faq` path and that fixed it.
